README.md

```diff
@@ -1,53 +1,35 @@
-# Welcome publishers!
 
-We’re glad you’re ready to learn about publishing your Web3 apps on the Solana dApp Store, the fee-free app store for the [Saga](https://solanamobile.com/saga) device.
+# dApp Publishing CLI
 
-What do we mean by fee-free? We mean that no fees are collected by Solana Mobile Inc. from users of the dApp Store. There may still be network transaction fees or app-specific fees associated with usage of the Solana dApp Store, or of apps installed by it. None of these fees are collected by, or on behalf of, Solana Mobile Inc.
+Tooling for publishing to the Solana Mobile dApp Store.
 
-Want to know more? Read more [about the Solana dApp Store](docs/ABOUT.md).
+For all documentation regarding usage of the tooling, including a thorough walkthrough of the dApp publishing process, visit the [Solana Mobile docs site](https://docs.solanamobile.com/dapp-publishing/intro).
 
-# Getting started
+# Installation
```

Comment:
These new instructions look quite a bit shorter than the old docs/SETUP.md. Is that content outdated, or moved elsewhere?